datastore/sqlite_datastore.nim

```diff
@@ -172,7 +172,7 @@ proc timestampCol*(
 
 proc new*(
   T: type SQLiteDatastore,
-  basePath = "data",
+  basePath = getCurrentDir(),
```

Comment:
Same question as in my previous comment.

Reply:
Based on discussion during today's team call, I have changed it so there's no default value for the `basePath` parameter.